### ISSUE TITLE
ETH support ?

### ISSUE THREAD

**Issue by Manosuper** [albertobsd/keyhunt#89]
Luis, good day.

Can you implement ETH support ?

BTC is very hard to crack - success 0 Bro ((((.

Regard.

**Comment by dem10** [comment]
You think ETH is easier to find the private key? Ether uses the same curve Secp256k1 as Bitcoin.And there are as many private keys on ETH as in bitcoin-fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364140.

**Comment by Manosuper** [comment]
I think ETH not sach more cracked and not cracked all posible keys. i think more interested is brute for part of privkey what prsiousy  SHA2(bitcoin) or SHA3(ETH). becaause crack direct SHA is mach more time expansive.....

**Comment by dem10** [comment]
BSGS works only with public keys. This is the shortest path private key-Secp256k1-public key. Each subsequent mathematical operation only slows down the tool.
![public-key-hash160](https://user-images.githubusercontent.com/24973894/125190811-af81b200-e247-11eb-9500-08d13b49c3d2.png)

**Comment by Manosuper** [comment]
ETH no such cracked like BTC....

**Comment by dem10** [comment]
Yes, the ether address is generated easier than that of bitcoin, but still the process of obtaining a public key is identical for both cryptocurrencies. As I wrote earlier, the public key is the fastest method. Everything else will be slower and there is no difference between bitcoin or ether, it is difficult to find a private key like in bitcoin.

**Comment by Manosuper** [comment]
ETH SSH generates from 12 word - so seed 128 bytes, bitcoin - 256 .

**Comment by albertobsd** [comment]
> ETH SSH generates from 12 word - so seed 128 bytes, bitcoin - 256 .

Really 128 bits only ? well anyway we still don't solve the 120 bit puzzle for bitcoin.

I am working in the update for ETH, can someone share some list of test for ETH address with the privatekey just to be sure that i generate correctly the address with the keccak algo.

Regards!

**Comment by albertobsd** [comment]
> If relevant, I can write you a utility for creating ETH addresses with private keys.

nice, thanks, well if you want to do that, go ahead, i only need some address list with the privatekeys just to validate the address that i am generating with the keyhunt tool :)

Regards!
